cookies/operations.py

```diff
@@ -172,3 +172,32 @@ def merge_resources(resources, master_id=None, delete=True, user=None):
     if delete:
         to_merge.delete()
     return master
+
+def add_resources_to_collection(resources, collection):
+    """
+    Adds selected resources to a collection.
+
+    Number of resources should be greater than or equal to 1.
+    And one collection has to be selected
+
+    Returns the collection after making changes.
+    """
+    if resources.count() < 1 :
+        raise RuntimeError("Need at least one resource to add to collection.")
+
+    if not collection:
```

Comment:
Good thinking to explicitly check these conditions first. For ``collection``, though, we know that this will be truthy unless the caller explicitly passes ``None``, ``False``, or ``0``. Might be more informative to check that ``collection`` is an instance of ``Collection``?